DAS/simulator.py

```diff
@@ -307,3 +311,89 @@ def run(self):
         self.result.populate(self.shape, self.config, missingVector)
         return self.result
 
+    def initDHTNetwork(self):
+        """ Compose the DHT network based on the pre-initialized Validators """
+        # compose the DHT networking layer
+        self.logger.info("Initializing DHTNetwork... with %d nodes" % self.shape.numberNodes, extra=self.format)
+        self.dhtNetwork = DHTNetwork(self.execID, self.shape.failureRate, [self.config.stepDuration])
```

Comment:
here the 3rd parame is `[self.config.stepDuration]`
but the underlying py-dht expects slowerrorrate:int
```
def __init__(self, networkid: int, fasterrorrate: int=0, slowerrorrate: int=0, conndelayrange = None, fastdelayrange = None, slowdelayrange = None, gammaoverhead: float = 0.0):
```

Reply:
I see similar issues in other places where DHTNetwork is initialized.